packages/frontend/yarn.lock

```diff
@@ -1103,6 +1103,259 @@
   resolved "https://registry.yarnpkg.com/@emotion/unitless/-/unitless-0.7.5.tgz#77211291c1900a700b8a78cfafda3160d76949ed"
   integrity sha512-OWORNpfjMsSSUBVrRBVGECkhWcULOAJz9ZW8uK9qgxD+87M7jHRcvh/A96XXNhXTLmKcoYSQtBEX7lHMO7YRwg==
 
+"@firebase/analytics-types@0.3.1":

```

Comment:
Czemu tu tyle zmian?

Reply:
czemu tyle paczek w yarn.lock a w package.jason nie widze zmian, podejrzane to

Reply:
nie mam pojęcia wlasnie

Reply:
Te zmiany wskakują tam po wykonaniu lerna clean i  lerna bootstrap